main/src/validator-resource-bundle.properties

```diff
@@ -0,0 +1,13 @@
+schema-error=Table Schema error: ${error_message}
+non-matching-header=Header in column ${column_number} doesn't match field name ${field_name} in the schema
+extra-header=There is an extra header in column ${column_number}
+missing-header=There is a missing header in column ${column_number}
+type-or-format-error=The value ${value} in row ${row_number} and column ${column_number} is not type ${field_type} and format ${field_format}
+required-constraint=Column ${column_number} is a required field, but row ${row_number} has no value
+pattern-constraint=The value ${value} in row ${row_number} and column ${column_number} does not conform to the pattern constraint of ${constraint}
+unique-constraint=Rows ${row_numbers} has unique constraint violation in column ${column_number}
+enumerable-constraint=The value ${value} in row ${row_number} and column ${column_number} does not conform to the given enumeration: ${constraint}
+minimum-constraint=The value ${value} in row ${row_number} and column ${column_number} does not conform to the minimum constraint of ${constraint}
+maximum-constraint=The value ${value} in row ${row_number} and column ${column_number} does not conform to the maximum constraint of ${constraint}
+minimum-length-constraint=The value ${value} in row ${row_number} and column ${column_number} does not conform to the minimum length constraint of ${constraint}
+maximum-length-constraint=The value ${value} in row ${row_number} and column ${column_number} does not conform to the maximum length constraint of ${constraint}
```

Comment:
How can we localize this file?

Reply:
Once we have i18n support on the server side